### PR TITLE
Added new hashes for the protobuf and py-protobuf packates.

### PR DESCRIPTION
**Pull request by bvanessen**

### CODE REVIEW

File: var/spack/repos/builtin/packages/protobuf/package.py [hunk 1]
```diff
@@ -15,7 +15,14 @@ class Protobuf(CMakePackage):
     url      = "https://github.com/protocolbuffers/protobuf/archive/v3.10.1.tar.gz"
     root_cmakelists_dir = "cmake"
 
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/protobuf/package.py]: This line is gonna cause flake8 to fail

**Comment by bvanessen** [comment]
I didn’t add that line, that was already there.

Brian C. Van Essen
vanessen1@llnl.gov
(w) 925-422-9300
(c) 925-290-5470

**Comment by adamjstewart** [comment]
But now there are 2 blank lines in a row, and before there was 1...

**Comment by bvanessen** [comment]
Ah, I fixed it.  Thanks.

**Pull request merged**